packages/cozy-pouch-link/src/helpers.spec.js

```diff
@@ -1,32 +1,59 @@
-import {
-  withoutDesignDocuments,
-  isDesignDocument,
-  isDeletedDocument
-} from './helpers.js'
+import { find, isDesignDocument, isDeletedDocument } from './helpers.js'
+import PouchDB from 'pouchdb'
+import PouchDBFind from 'pouchdb-find'
+import adapter from 'pouchdb-adapter-memory'
+PouchDB.plugin(PouchDBFind)
+PouchDB.plugin(adapter)
+
+const insertData = async (db, number = 2000) => {
+  for (let i = 0, time = new Date().getTime(); i < number; i++, time--) {
+    const doc = { _id: `doc${i}` }
+    try {
+      await db.put(doc)
+    } catch (e) {
+      console.log(`error(${doc._id}): ${e}`)
+    }
```

Comment:
A batch insert would be more performant : https://pouchdb.com/guides/bulk-operations.html#use-bulkdocs-to-write-many-docs